src/App.js

```diff
@@ -1,24 +1,100 @@
-import logo from './logo.svg';
+
+import { useState } from 'react';
+import contactsArr from "./contacts.json";
+
 import './App.css';
 
+
+
 function App() {
+  let fiveContacts = contactsArr.slice(0,5);
+
+  const [contacts, setContacts] = useState(fiveContacts);
+
+  const addRandomContact = () => {
+  if(!contactsArr.length){ return false;}
```

Comment:
also in contactArr werden ja immer Kontakte sein, daher brauchst du diese if Bedingung hier nicht